src/param.rs

```diff
@@ -2885,7 +2801,227 @@ mod tests {
         param.set_value_at_time(2., 0.000001);
         param.set_automation_rate(AutomationRate::A);
 
-        let vs = render.tick(0., 1., 10);
+        let vs = render.compute_intrisic_values(0., 1., 10);
         assert_float_eq!(vs, &[2.; 10][..], abs_all <= 0.);
     }
+
+    #[test]
+    fn test_computed_values_clamped() {
```

Comment:
All these are a bit redundant, I will try to come up w/ something more simple tomorrow